src/v2/features/vs/damage-calculator-service.tsx

```
@@ -0,0 +1,127 @@
+                return 70;
+        }
+        return 0;
+    }
+
+    private static convertDamageType(type: string | undefined): DamageType | undefined {
+        if (type == undefined) return undefined;
+        return DamageType[type as keyof typeof DamageType];
+    }
+
+    private static getDamageFrom1Hit(damage: number, type: DamageType, armor: number, gravis: boolean): number {
+        const damageWithPierce = (damage * this.getPierce(type)) / 100;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)